lib/code_corps/messages/conversation_query.ex

```diff
@@ -41,8 +41,8 @@ defmodule CodeCorps.Messages.ConversationQuery do
   - belong to a `CodeCorps.Message` initiated by admin, with at least a single
```

Comment:
We need to update this documentation.

> Filters `CodeCorps.Conversation` record queries to return only those considered to be active.
>
> Active conversations belong either:

And then update the bullets to remove "belong".